app/desmos/cmd/testnet.go

```diff
@@ -56,7 +59,10 @@ Example:
 	desmos testnet --v 4 --output-dir ./output --starting-ip-address 192.168.10.2
 	`,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			clientCtx := client.GetClientContextFromCmd(cmd)
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
 
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
```

Comment:
It can't be run on my local since the state-sync in `config.toml` is enabled as default. Can you add `config.StateSync.Enable = false` to disable testnet statesync here?

Reply:
@dadamu Can you give me more details please? What does not work? The chain doesn't start? The upgrade doesn't happen? Or what else? Cause if me state sync enabled is giving problems with the upgrade, we need to disable it by default reverting your changes and open a Cosmos SDK issue immediately

Reply:
@RiccardoM The chains doesn't start because it is trying to state-sync, It is solved after adding the `config.StateSync.Enable = false` line into `testnet` command. Except for it, everything works properly.

Reply:
The error log of the container is: 
```
1:38PM INF Starting PEX service impl=PEX module=pex
1:38PM INF Starting AddrBook service book=/desmos/node1/desmos/config/addrbook.json impl=AddrBook module=p2p
1:38PM INF Starting Mempool service impl=Mempool module=mempool
1:38PM ERR Can't add peer's address to addrbook err="Cannot add non-routable address 24cc5ad822378faa995c56728789897bc733ab7b@192.168.10.2:26656" module=p2p
1:38PM ERR Can't add peer's address to addrbook err="Cannot add non-routable address 624e48e0d322652c90bcc5c9a473fca15a00c8d6@192.168.10.5:26656" module=p2p
1:38PM ERR Can't add peer's address to addrbook err="Cannot add non-routable address 6ba80376c2c549a7e2da5e856736f258ed4d27e8@192.168.10.4:26656" module=p2p
1:38PM INF Saving AddrBook to file book=/desmos/node1/desmos/config/addrbook.json module=p2p size=0
1:38PM INF Ensure peers module=pex numDialing=0 numInPeers=0 numOutPeers=0 numToDial=10
1:38PM INF No addresses to dial. Falling back to seeds module=pex
1:38PM INF Starting state sync module=statesync
Error: failed to start state sync: failed to set up light client state provider: at least 2 RPC servers are required, got 0
```

Reply:
@RiccardoM I think I found the reason. It is my bad. The error happens since the `testnet` command copies my local "~/.desmos" config to generated testnet folders. So, It works fine after removing "~/.desmos".